### DISCUSSION TITLE
Weird error while using inline variable

### DISCUSSION THREAD

**Discussion by anilgoel** [artilleryio/artillery#991] [Artillery]
I am using very simple command to test artillery script and trying to pass variable from the command line. Command is

artillery run  -v '{"var1": "value1", "var2": "value2"}' .\test\performance\applications.yml -e development

It is giving error

Variable definition is not valid JSON. Correct example: -v '{"var1": "value1", "var2": "value2"}'

I cannot see any difference between my command and the error. Any help??

**Comment by hassy** [comment]
That's a known quoting issues on Windows, please see: https://github.com/artilleryio/artillery/issues/984
